ShapeAnalysis/python/PlotFactory.py

```diff
@@ -1159,7 +1212,8 @@ def makePlot(self, inputFile, outputDirPlots, variables, cuts, samples, plot, nu
 
                 # If dividing by bin width, yaxis should be "<Events / [unit]>"
                 if variable["divideByBinWidth"] == 1:
-                    frameDistro.GetYaxis().SetTitle("< Events / %s >"%unit)
+                    # frameDistro.GetYaxis().SetTitle("< Events / %s >"%unit)
+                    frameDistro.GetYaxis().SetTitle("dN/d"+variable['xaxis'])
```

Comment:
from SMP-20-013 review. Maybe others prefers the <Events / > label

Reply:
Can we have this configurable with an option "1, 2, 3, ..." that will use different styles?

Reply:
would you suggest a name for this option?

Reply:
if this is the only holdback, maybe we can merge it as is and tweak it later

Reply:
You are right, let me revert my change.